pkg/process/checks/process_test.go

```diff
@@ -117,6 +123,12 @@ func TestProcessCheckSecondRun(t *testing.T) {
 	proc3 := makeProcessWithCreateTime(3, "foo --version", now+2)
 	proc4 := makeProcessWithCreateTime(4, "foo -bar -bim", now+3)
 	proc5 := makeProcessWithCreateTime(5, "datadog-process-agent --cfgpath datadog.conf", now+2)
+
+	proc1.ProcessContext = []string{"process_context:git"}
+	proc2.ProcessContext = []string{"process_context:mine-bitcoins"}
+	proc3.ProcessContext = []string{"process_context:foo"}
+	proc4.ProcessContext = []string{"process_context:foo"}
+	proc5.ProcessContext = []string{"process_context:datadog-process-agent"}
```

Comment:
Right now you have two different approaches (`makeProcessWithProcessContext` or just assigning the field) for adding the process context to procs. I think you should choose one way for consistency. I think what makes most sense is updating `makeProcessWithCreateTime` to accept a context parameter and set it as it seems its always needed.

Reply:
Decided to not do either and go with an approach where procutil does not have to be modified